source/iterators.tex

```diff
@@ -1542,8 +1542,8 @@
 
 \pnum
 The \libconcept{sized_sentinel_for} concept specifies
-requirements on an \libconcept{input_or_output_iterator} and
-a corresponding \libconcept{sentinel_for}
+requirements on an \libconcept{input_or_output_iterator} type \tcode{I} and
```

Comment:
Do you think I should've used math mode _I_ instead?

Reply:
Not right now in this isolated instance.